### ISSUE TITLE
[BUG] - Backdrop blur being applied on animation end

### ISSUE THREAD

**Issue by simonetimi** [nextui-org/nextui#3482]
### NextUI Version

2.4.2

### Describe the bug

On the Dropdown component, applying a custom background works as expected. However, when applying a tailwind class like backdrop-blur-lg, the effect is applied only at the end of the opening animation, unlike other classes like bg-color.

In my case, I'm trying to make the style consistent with the rest of the UI. Having some semi-transparent UI elements, blur is necessary to keep the text readable. Having it switch to blur only after the animation ends makes it unpleasant. For now I'm disabling the animation but it's obviously not ideal.

### Steps to Reproduce the Bug or Issue

1. Add backdrop blur tailwind class to a Dropdown component
2. Click on the dropdown trigger

### JSX code in client component
```jsx
<Dropdown
    className="bg-white/40 backdrop-blur-lg dark:bg-black/40"
>
    <DropdownTrigger>
        <Button isIconOnly className="bg-transparent" disableRipple>
            <EllipsisVerticalIcon width={20} />
        </Button>
    </DropdownTrigger>
    <DropdownMenu aria-label="Todo Actions">
        <DropdownItem key="details" onPress={onOpen}>
            Show details
        </DropdownItem>
        <DropdownItem key="edit">Edit</DropdownItem>
        <DropdownItem
            key="delete"
            className="text-danger"
            color="danger"
        >
            Delete
        </DropdownItem>
    </DropdownMenu>
</Dropdown>
```

### Expected behavior

The blur effect should be displayed consistently with the other styles, at the beginning of the animation.

###  During the animation
<img width="311" alt="Screenshot 2024-07-16 alle 13 09 18" src="https://github.com/user-attachments/assets/837d6284-038f-43e4-a61e-0fc049a95780">

### After the animation
<img width="288" alt="Screenshot 2024-07-16 alle 13 09 27" src="https://github.com/user-attachments/assets/332b5703-1608-4e35-8e37-ccb3e15d913b">

### Operating System Version

macOS

### Browser

Chrome

**Comment by Tejas-Sands** [comment]
there is a property in dropdown called backdrop , in which you can select blur option 
it doesn't have the animation problem but blurs out the background too

**Comment by simonetimi** [comment]
> there is a property in dropdown called backdrop , in which you can select blur option it doesn't have the animation problem but blurs out the background too

That option will have an effect on the whole page (expect for the menu), I'm looking for the exact opposite (I need the menu background to be blurred). So far I can either use a solid background (which unfortunately doesn't match with the rest of the UI) or disable the animation.

**Comment by Tejas-Sands** [comment]
I think this problem occurs because backdrop property is added in pop over prop
Instead it should have been added to the menu prop

**Comment by abhinandan-verma** [comment]
@simonetimi  can you minimal reproducible enviroment like CodeSankBox or StackBlitz

**Comment by simonetimi** [comment]
> @simonetimi can you minimal reproducible enviroment like CodeSankBox or StackBlitz

I set it up on StackBlitz: [sandbox](https://stackblitz.com/edit/stackblitz-starters-pjxozs?file=app%2Fpage.tsx)

I put some text underneath. When clicking the button, the blur on the menu bg will show only when the animation ends. The sandbox doesn't work for me in Safari for some reason, but this bug is not present in safari anyway.

**Comment by abhinandan-verma** [comment]
@simonetimi  I observed you issue, the effect is only visible in `Chrome`. I am working on it and will help you soon

**Comment by ritik3236** [comment]
any updates?

**Comment by ritik3236** [comment]
6 months, 1 week and 2 days passed still no update ?

**Comment by amandakelake** [comment]
Mark, wait for updates

**Comment by kzroo** [comment]
yeah the reason why is because of the animation from framer-motion.

So to fix this, pass the motion props and add the backdrop-blur style to it like this:

```typescript
<Dropdown motionProps={{
                    style: {
                        backdropFilter: "blur(var(--blur-lg)) saturate(150%)",
                        borderRadius: "var(--radius-xl)",
                    }
                }}
... rest of your code
``` 

OR, just put `disableAnimation` prop on your dropdown.

**Comment by wingkwong** [comment]
Closing in favor of v3. If the issue is reproducible in v3, please create a new one with a minimal reproducible example.
